dbms/src/Storages/Page/V3/PageDirectory.cpp

```diff
@@ -473,7 +473,7 @@ std::optional<PageEntryV3> VersionedPageEntries<Trait>::getLastEntry(std::option
     auto page_lock = acquireLock();
     if (type == EditRecordType::VAR_ENTRY)
     {
-        for (auto it_r = entries.rbegin(); it_r != entries.rend(); it_r++)
+        for (const auto & entrie : std::ranges::reverse_view(entries))
```

Comment:
```suggestion
        for (auto it_r = entries.rbegin(); it_r != entries.rend(); it_r++)
```
keep it not change because we don't support clang 15 in CI by now